### DISCUSSION TITLE
Add support for jami

### DISCUSSION THREAD

**Discussion by Allfifthstuning** [d99kris/nchat#223] [Ideas]
Would be nice if jami could also be added.
There's a cli client (https://github.com/amarok1412/jami-cli) but it would be nice if there'll be `one ring to rule them all".

https://jami.net/

**Comment by d99kris** [comment]
Thanks for suggesting a feature. As the current sole maintainer of nchat and not a Jami user myself, it's unlikely this will get added. There have been other requests, for example #79 for nostr, which are not considered for the same reason. 

But like mentioned on that other feature request; if someone would be willing to contribute implementation for it, and can help with related bugs, I would be open to consider including it in the nchat code base. However, the likelihood of someone contributing that is probably low.
